src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -5,13 +5,8 @@
 			:icon-key="promoBannerContent.iconKey"
 			:promo-banner-content="promoBannerContent"
 		/>
-		<appeal-banner
-			v-if="showAppeal"
-			:appeal-banner-content="appealBannerContent.fields"
-		/>
-		<appeal-banner-15
-			v-if="show15YearAppeal"
-			:cta-link="appeal15link"
+		<appeal-banner-swashie-container
+			v-if="customAppealEnabled"
```

Comment:
Please promote this new swashie face banner to be the defacto "Appeal Banner" both here and in Contentful. The "Custom Appeal" type should be for more one-off endeavors such as the Kiva 15.